nengo/builder/probe.py

```diff
@@ -33,10 +33,8 @@ def synapse_probe(model, key, probe):
         raise ValueError("Attribute '%s' is not probable on %s."
                          % (key, probe.obj))
 
-    if isinstance(probe.slice, slice):
+    if probe.slice is not None:
         sig = sig[probe.slice]
-    else:
```

Comment:
Ah alright, we can put the warning back in then.  Or I guess we could implement it, but that seems like more work :sweat: